org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/ConstructorDelegation.wlk.xt

```diff
@@ -92,14 +92,14 @@ class Superclase {
 class InvalidInstanceAccessOnConstructorDelegation inherits Superclase {
 	var x = 1
 	
-	// XPECT errors --> "Cannot access instance methods within constructor delegation." at "self"
+	// XPECT warnings --> "Cannot access instance methods within constructor delegation." at "self"
```

Comment:
Si es un warning, el mensaje de error no debería ser "Cannot". Podría ser "Accessing instance methods within constructor delegation is dangerous because object might not be completely initialized".

Reply:
fixed